reactor-error-prone/src/test/java/com/github/lhotari/reactor/errorprone/ReactorInnerPublisherIgnoredTest.java

```diff
@@ -42,7 +42,7 @@ public void reactorFluxOfFlux() {
                         "  }",
                         "  Mono<Void> and() {",
                         "    // BUG: Diagnostic contains: The inner Flux|Mono (Publisher) is ignored and it is never scheduled for execution",
-                        "    return getFlux().and(null);",
+                        "    return getFlux().then(null);",
```

Comment:
I replaced all occurrences of `and(null)` with `then(null)`, since `and` doesn't exist. Let me know if this was meant to represent another method.